File: old/ClienteDesktop/GoTravel/src/main/java/com/gotravel/cliente/model/Paypal.java
package com.gotravel.cliente.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "paypal")
public class Paypal extends Metodopago {

    @NonNull
    @Column(name = "Email", nullable = false, length = 45)
    private String email;

}

File: GoTravelMobile/app/src/main/java/com/gotravel/mobile/data/model/Paypal.kt
package com.gotravel.mobile.data.model

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
data class Paypal(
    override val id: Int?,
    val email: String
) : Metodopago(id)